examples/meta/src/kernel/cauchy.sg.in

```diff
@@ -7,9 +7,7 @@ Features feats_test = create_features(f_feats_test)
 #![create_features]
 
 #![create_kernel]
-Distance dist = create_distance("EuclideanDistance")
-dist.init(feats_train, feats_train)
-Kernel cauchy = create_kernel("CauchyKernel", sigma=1.0, distance=dist)
+Kernel cauchy = create_kernel("CauchyKernel", sigma=1.0)
```

Comment:
you can still pass along the distance object explicitly no?

Reply:
Of course, but what's to stop the users to using a distance which isn't aligned with the formula? 

For ex: Circular uses L1 as per formula but users can use L2 too.

 Wouldn't that lead to confusion?